Core/src/org/sleuthkit/autopsy/modules/embeddedfileextractor/ImageExtractor.java

```diff
@@ -211,6 +213,13 @@ void extractImage(AbstractFile abstractFile) {
         HWPFDocument doc = null;
         try {
             doc = new HWPFDocument(new ReadContentInputStream(af));
+        } catch (OldFileFormatException ex) {
```

Comment:
I did another commit to remove the rest of these, and then I did a second commit to remove an unused error from the Bundle.properties file.